acquia_lift.context.inc

```diff
@@ -75,9 +75,14 @@ function _acquia_lift_get_page_context($node = NULL, $primary = FALSE) {
 
   // Add the content type of the page if this is the primary node displayed.
   if ($primary && empty($page_context['post_id'])) {
+    // Get global language to set as 'context_langauge'
+    // Note: Language will always be set as there is a default fallback
+    global $language;
+
     $page_context['content_type'] = $node->type;
     $page_context['content_title'] = $node->title;
     $page_context['published_date'] = $node->created;
+    $page_context['context_language'] = $language->language;
```

Comment:
New code on language should probably be outside of this if statement, so `context_language` is surfaced to markup on non-node pages (e.g. home page).

Reply:
added outside of statement